docs/.docz/app/db.json

```diff
@@ -3,11 +3,16 @@
     "title": "Warthog",
```

Comment:
Does this typically get committed or should this only be built and uploaded and not stored in source control?